lib/redcord/errors.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require 'redcord/errors'
+
+module Redcord
```

Comment:
Is it worth having these specific errors subclass a `RedcordError` (or whatever name) instead of `StandardError`? It could make error handling easier in situations where you want to rescue all errors from Redcord.

Reply:
I think that'd definitely be an improvement. I'll open a separate issue in this repo https://github.com/chanzuckerberg/redcord/issues/100